internal/parsers/table/csv/calls.go

```diff
@@ -79,16 +81,38 @@ func (t *Table) SubDivName(countryCode *locodedb.CountryCode, code string) (stri
 	return rec.name, nil
 }
 
+func utf8encoding(subdivCountry string) (string, error) {
+	res, err := charmap.Windows1256.NewDecoder().String(subdivCountry)
+	if err == nil && !strings.Contains(res, "\uFFFD") && !strings.Contains(res, "\u0000") && !strings.Contains(res, "?") {
+		return res, nil
+	}
+
+	res, err = charmap.ISO8859_1.NewDecoder().String(subdivCountry)
+	if err == nil && !strings.Contains(res, "\uFFFD") && !strings.Contains(res, "\u0000") {
+		return res, nil
+	}
+
+	return subdivCountry, errors.New("could not convert to res")
+}
+
 func (t *Table) initSubDiv() (err error) {
 	t.subDivOnce.Do(func() {
 		t.mSubDiv = make(map[subDivKey]subDivRecord)
 
 		err = t.scanWords([]string{t.subDivPath}, subDivFldNum, func(words []string) error {
+			subdiv := words[subDivName]
+			if !utf8.ValidString(subdiv) {
+				subdiv, err = utf8encoding(subdiv)
+				if err != nil {
+					return err
```

Comment:
So the process will fail? Or we're going to have some warning printed for this record? What's gonna happen?

Reply:
process will fail

Reply:
But we have to have some DB in the end, don't we?

Reply:
changed to printing warning about the record